src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -156,6 +160,10 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
             buildImageCmd = buildImageCmd.withBuildArg(arg, value)
         }
 
+        if (!cacheFrom.isEmpty()) {
```

Comment:
You should be able to get away with `if (cacheFrom) { ...` as groovy's truthiness check should take into account empty empty collections if I'm not mistaken.

Reply:
For the time that we still use convention mapping we should rather use the getter methods instead of the direct access to the property. For example see this: https://github.com/exFalso/gradle-docker-plugin/blob/f648a028dccd83a909571fbc983e55f3eb694e69/src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy#L153-L154